flink-table/flink-table-planner/src/test/resources/org/apache/flink/table/planner/plan/batch/sql/join/LookupJoinTest.xml

```diff
@@ -663,7 +663,7 @@ Calc(select=[EXPR$0, EXPR$1, EXPR$2])
    +- Exchange(distribution=[hash[b, b0]])
       +- LocalHashAggregate(groupBy=[b, b0], select=[b, b0, Partial_COUNT(a) AS count$0, Partial_COUNT(id) AS count$1, Partial_SUM(a0) AS sum$2])
          +- HashJoin(joinType=[InnerJoin], where=[(a = a0)], select=[b, a, id, a0, b0], build=[right])
-            :- Exchange(distribution=[hash[a]], shuffle_mode=[BATCH])
```

Comment:
I find the change in BatchTestBase:
![image](https://user-images.githubusercontent.com/8777671/215318877-7cd866da-0e8b-4bff-8c2d-b7fc7b83d338.png)

Reply:
Yes, all the tests which only tests ALL_EXCHANGES_PIPELINED are changed to test ALL_EXCHANGES_BLOCKING because the latter is the default value and is widely used in production.